### PR TITLE
Following routes

### PR DESCRIPTION
**Pull request by aitchslash**
A post route to add a new follow.
A get route that returns all users that a User is following.
More to come.

### CODE REVIEW

**Review by dipshojol — CHANGES_REQUESTED** [review]
file posts.js need to be updated. take the latest

File: server/routes/posts.js [hunk 1]
```diff
@@ -47,13 +48,51 @@ router.get("/discovery", function(req, res) {
...
+    .catch(err => res.status(400).send({ error: err }));
+});
+
+router.get("/fx", function(req, res) {
+  // res.send({ success: "pooong." });
+  Post.find({ author: "5dd1394a7025091ac840b579" }).then(result =>
+    res.send(result)
+  );
+});
+
```
Comment (dipshojol) [inline server/routes/posts.js:95]: dont see the delete route???

**Review by dipshojol — CHANGES_REQUESTED** [review]

File: server/routes/posts.js (hunk 1 above)
Comment (aitchslash) [reply server/routes/posts.js:95]: Good spot!  I haven't written it yet.

**Comment by aitchslash** [comment]
> file posts.js need to be updated. take the latest

Please refer to some code.

**Pull request merged**